raiden/tests/test_accounts.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+import os
+import pytest
+
+from ethereum.utils import encode_hex
+from raiden.accounts import AccountManager
+from raiden.utils import get_project_root
+
+test_keystore = os.path.join(get_project_root(), 'tests', 'test_files')
+
+
+def test_get_accounts():
+    accmgr = AccountManager(test_keystore)
```

Comment:
haha ... okay point taken. I actually also tend to make long descriptive names for classes, not so much for short-lived variables that get assigned to from a class with a descriptive name.